src/pages-and-resources/app-settings-modal/AppSettingsModal.jsx

```diff
@@ -243,6 +244,7 @@ const AppSettingsModal = ({
                   )}
                 />
               )}
+              {bodyChildren}
```

Comment:
I need to render the two OpenAI links. I'm copying these links from the Xpert unit summaries settings modal. However, I didn't want to follow the pattern of copying and pasting the `AppSettingsModal` into the Learning Assistant component, so I thought this would be a better approach.